test/simplify_algebra_test.cpp

```diff
@@ -734,6 +761,157 @@ TEST_CASE(simplify_div_const)
     EXPECT(m1 == m2);
 }
 
+TEST_CASE(simplify_unit_mult_const)
+{
+    migraphx::module m1;
+    {
+        auto x    = m1.add_parameter("x", {migraphx::shape::int32_type, {1}});
+        auto unit = m1.add_literal(1);
+        m1.add_instruction(migraphx::make_op("mul"), x, unit);
+    }
```

Comment:
It would be good to have tests using multidimensional tensors(which is much more common) rather than just scalars.

Reply:
Makes sense, I suppose both would be valid then to verify that we cover all our bases then.